### DISCUSSION TITLE
脱出ホラーゲーム（仮）の必要要件

### DISCUSSION THREAD

**Discussion by saito-simaru** [EyeTracker-123/UnityProject#9] [Ideas]
ざっくりと必要要件（コアとなるシステム）考えてみた

・プレイヤーの制御
・鬼の制御
・目の開閉で音量変更
・ステージやオブジェクトに対するアクション

**Comment by OzuAkira** [comment]
> ・プレイヤーの制御 
> ・鬼の制御
> ・目の開閉で音量変更 
> ・ステージやオブジェクトに対するアクション

これをもとに私の方でIssues入れておきますね

**Comment by saito-simaru** [comment]
Issueの登録ありがとうございます。

今後の作業についてですが、
作業内容を被らせたくないのでozさんには引き続きプレイヤーの制御を担当していただきたいです。

私はオブジェクト関連を進めてみます。

KeitaはんはUnity事始めで3Dの基礎を知っていただければと思います。（数学できる人は心強い）

**Reply by Keita062** [reply]
3Dについての理解深めるの了解。
かつ、三角関数、ベクトルの軽い復習行います。
